src/libprojectM/projectM.cpp

```diff
@@ -218,7 +218,7 @@ void projectM::readConfig (const std::string & configFile )
     projectM_init ( _settings.meshX, _settings.meshY, _settings.fps,
                     _settings.textureSize, _settings.windowWidth,_settings.windowHeight);
 
-    _settings.beatSensitivity = beatDetect->beat_sensitivity = config.read<float> ( "Hard Cut Sensitivity", 10.0 );
+    _settings.beatSensitivity = beatDetect->beat_sensitivity = config.read<float> ( "Hard Cut Sensitivity", 1.0 );
```

Comment:
I feel like "Hard Cut Sensitivity" should be changed

Reply:
> I feel like "Hard Cut Sensitivity" should be changed

I would agree. I can rename it to Beat Sensitivity and update configs. 

It seems Milkdrop used to have Soft Cuts and Hard Cuts for transitions (I didn't know this).

- Soft cuts were more frequent and based on a duration (default 16 seconds) + some randomness (up to 10 seconds by default).
- Hard cuts were less frequent and based on a loudness event **(disabled by default).**

My guess is hard cuts aren't needed as they were disabled by default anyway.

![image](https://user-images.githubusercontent.com/59471060/80993124-96af8c80-8e4b-11ea-9bea-e87813aef3f9.png)

Reply:
I did more digging and it seems projectM supports soft cuts and hard cuts, and it's doing them both.

Right now the hard cut duration is fixed to 3 (HARD_CUT_DELAY) and timekeeper is checking if we passed 3 seconds with a check on the difference between current volume vs. the previous volume is greater than the Hard Cut Sensitivity (what we now call Beat Sensitivity).

`else if ((beatDetect->vol-beatDetect->vol_old>beatDetect->beat_sensitivity ) &&
                 timeKeeper->CanHardCut())`

I recommend:
- Keep Hard Cut Sensitivity and document what it is. I could also make it a number between 0 to 100 but on my system volume is normally only showing as a number very low. I would like feedback from others once they start testing the stats PR.
- Add a new variable for Hard Cut Duration (default 60 seconds).
- Add a new variable / setting for Beat Sensitivity.

Reply:
👍 please document your findings!